realm/realm-library/build.gradle

```diff
@@ -46,6 +46,7 @@ repositories {
 
 dependencies {
     provided 'io.reactivex:rxjava:1.1.0'
+    compile "com.android.support:recyclerview-v7:23.1.1"
```

Comment:
Hmm, that is pretty annoying. We will need to discuss this internally I think. Do you know how big the recyclerview dependency is in terms of size/method count?